src/taco-utils/tacoPackageLoader.ts

```diff
@@ -322,7 +322,7 @@ module TacoUtility {
         private static createTacoPackageInstallRequest(packageKey: string, dependencyConfigPath: string, logLevel: InstallLogLevel): IPackageInstallRequest {
             if (fs.existsSync(dependencyConfigPath)) {
                 try {
-                    var dependencyLookup: any = require(dependencyConfigPath);
+                    var dependencyLookup: any = JSON.parse(<any> fs.readFileSync(dependencyConfigPath));
```

Comment:
We have a lot of JSON.parse's throughout the code with similar workarounds already